Create PersonasBTCPay.md

pavlenex opened the pull request:
Very early draft. Consider it as a bunch of random ideas I've tried to document.
Needs proper feedback if this is a direction we want to go in, and possibility an idea of how to [format it.](https://github.com/peakshift/bitcoin-ux/blob/master/coinjoins/design-sprints.md#persona)

johnsBeharry commented on merchants/Personas/PersonasBTCPay.md:
```diff
@@ -0,0 +1,44 @@
+Usually an organization that's trying to raise funding for a particular cause, or receive donations in Bitcoin. Larger organizations usually convert their donations to fiat by exchanging their bitcoin on an exchange. These charities have deep understanding of privacy and care about privacy. They don't pay attention to invoices, underpaid amounts or manage their invoices within BTCPay Server. They ...
+Example of Charity persona: [The Tor Project](https://donate.torproject.org/cryptocurrency/), [Human Rights Foundation](https://hrf.org/donate-bitcoin/), [Hong Kong Free Press](https://support.hongkongfp.com/)
+
+#### Censored merchants
```
I don't know if such merchants exist, but I can imagine in some places accepting money online is difficult - or not possible because of the lack of access to merchant accounts, not due to intentional censorship.

Which of these archetypes would such merchants will fall into?

pavlenex replied:
That's a great idea. I would probably form a new group for those type of users, because they don't care probably about the privacy and censorship resistance, they simply do not have any other options. I guess it can in a way be sub-group in Censored merchants, or do you have any other ideas?

johnsBeharry commented on merchants/Personas/PersonasBTCPay.md:
```diff
@@ -0,0 +1,44 @@
+
+#### Charity
+
+Usually an organization that's trying to raise funding for a particular cause, or receive donations in Bitcoin. Larger organizations usually convert their donations to fiat by exchanging their bitcoin on an exchange. These charities have deep understanding of privacy and care about privacy. They don't pay attention to invoices, underpaid amounts or manage their invoices within BTCPay Server. They ...
```
In 2018, just as the large earthquake hit Indonesia there were a bunch of small teams popping up to assist with the relief efforts. Bitcoin was sent from overseas, and converted at IDR using a local exchange because of the speed.

These teams were not formal non-profits, just people willing to help and did not have an organisation to have wire transfers sent to.

No web pages existed until later as aftershocks and other smaller earthquakes were going on, so was mostly done through mobile wallets, social media and chat apps from friends and family overseas who were wanting to help financially.

pavlenex commented on merchants/Personas/PersonasBTCPay.md:
```diff
@@ -0,0 +1,44 @@
+Bitcoin enthusiasts usually deploys his online store for fun and experiment. These users have technical skills and want to try BTCPay Server out. These are small shops or attempts to collect Bitcoin within a certain type of group (Bitcoin Twitter).
+
+#### Medium-size merchants
+These merchants use CMS in most of the cases for their online stores. They use BTCPay via available plugins and easily connect it to their stores.
```
Usually it's set and forget for online merchants, unless there's a problem. In that case if they go to Invoices and do a query for order id, email, name to identify the invoice and then troubleshoot. It occurred to me that we can probably focus on troubleshooting and making that easier.

pavlenex commented:
@johnsBeharry as agreed, I split buyer vs merchant's invoices and added an early draft of buyer's persona, but need to think about it more. Will merge for now, so we don't lose scope due to frequent PR changes I've made.
